docs/source/getting_started.rst

```diff
@@ -1,154 +1,217 @@
+It is important to know that the ``smaht-submitr`` tool and SMaHT will ensure that the referenced
+objects actually exist within the SMaHT Portal, `or` are defined within the spreadsheet itself;
+if this is not the case then an error will be the result.
+The identifying value for an object varies depending on the specific object in question,
+though the ``uuid`` is common to all objects; other common identifying properties
+are ``submitted_id`` and ``accession``.
 
-   * - Object
-     - Field
-     - type/ID
-     - ID
-   * - Consortia
-     - identifier
-     - /consortia/smaht-consortia/
-     - smaht-consortia
-   * - Submission Center
-     - identifier
-     - /submission-center/dac/
-     - dac
-   * - User
-     - email
-     - /users/test@test.com/
-     - test@test.com
+|
 
+Submission
+----------
 
+The type of submission supported is called a "metadata bundles", or `accessioning`.
+And the name of the command-line tool to initiate a submission is ``submit-metadata-bundle``.
+A brief tour of this command, its arguments, and function is described below.
 
 Getting Added as a SMaHT User or Submitter
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Before you can view protected data or submit data to the SMaHT system you must be a registered user of the site and have the appropriate access credentials.
+The ``smaht-submitr`` tool is a Python based command-line tool, distributed
+on the `Python Package Index <https://pypi.org/project/smaht-submitr/>`_ which can be installed
+using the standard Python ``pip`` utility like this::
+
+    pip install smaht-submitr
+
+For more details on installation process please see `here <https://submitr.readthedocs.io/en/latest/>`_.
+
+**Basic Usage**
+
+The basic ``smaht-submitr`` command is called ``submit-metadata-bundle``. To get help about this command, do::
+
+   submit-metadata-bundle --help
+
+For many cases it will suffice simply to specify the metadata bundle file you want to upload,
+and the SMaHT environment name (such as ``data`` or ``staging``) from your ``~/.smaht-keys.json`` keys file).
+For example::
+
+   submit-metadata-bundle your_metadata_file.xlsx --env data
+
+You can omit the ``--env`` option entirely if your ``~/.smaht-keys.json`` file has only one entry.
+
+This command should do everything, including uploading referenced file; it will prompt first for confirmation;
+see the `Uploading Referenced Files` section just below for more on this.
+
+If you belong to
+multiple consortia and/or submission centers, you can also add the ``--consortium <consortium>``
+and ``--submission-center <submission-center>`` options; if you belong to only one of either,
+the command will automatically detect (based on your user profile) and use those.
+
+**Sanity Checking**
+
+To invoke the submission for with `local` sanity checking, where "local" means - `before` actually submitting to SMaHT, do::
+
+   submit-metadata-bundle your_metadata_file.xlsx --env <environment-name> --check
```

Comment:
We should probably default this or include an example that shows `data` since this is the only valid option for users